CI/appveyor.functions.ps1

```diff
@@ -254,7 +255,7 @@ function InstallHunspell() {
 }
 
 function InstallYajl() {
-  $Env:Path = $NoShPath
+  $Env:Path = filterPathForSh($ShPath)

```

Comment:
The function does not take an argument...

Reply:
Yes, I have no idea what I was thinking with these changes. Thea actual problem is that I have a null path entry, for ex. `PATH="/bin;;/bin/bash`. Notice the `;;`. With powershell `Path-Join` it will fail here but the Combine functionality seems to do just fine.

If there's a better way to handle this let me know, first time working with powershell.